include/eventFunctions.php

```diff
@@ -103,9 +109,14 @@ function echoEvent($event){
 
 	echo"
 		<div class='voteColumn'>
-			<p> <img class='iconAligned' style='background-color:white' src='/pics/arrow2.jpg' alt='arrow' height=40px></p>
-			$event[votes]
-			<p> <img class='iconAligned' style='background-color:white' src='/pics/line2.jpg' alt='line' height=40px></p>
+			<br>
+			<a href='#' onclick='upVoteNumber();'><img class='iconAligned' style='background-color:white' src='/pics/arrow2.jpg' alt='arrow' height=40px></a>
```

Comment:
You said you were having problems getting this upvote working. You can solve this by passing in an EventId to the `upVoteNumber` function.

While you're at it, here are some things you could do to make this code a bit more general and re-usable:

1) Make a PHP function for echoing the upvote and downvote buttons. That function takes an EventId, and then you can re-use it on both of your pages that display events.

2) Make a js file that contains the upvote and downvote functions. Make sure they're written in a way where they'd work on both pages. Include that js file on both pages so you can reference the same functions from both places.